lib/std/ioutils.nim

```diff
@@ -0,0 +1,50 @@
+when defined(windows):
+  proc c_dup(oldfd: FileHandle): FileHandle {.
+    importc:"_dup", header: "<io.h>".}
+  proc c_dup2(oldfd: FileHandle, newfd: FileHandle): cint {.
+    importc: "_dup2", header: "<io.h>".}
+else:
+  proc c_dup(oldfd: FileHandle): FileHandle{.
+    importc: "dup", header: "<unistd.h>".}
+  proc c_dup2(oldfd: FileHandle, newfd: FileHandle): cint {.
+    importc: "dup2", header: "<unistd.h>".}
+
+when false:
+  const SupportIoctlInheritCtl = (defined(linux) or defined(bsd)) and
+                                not defined(nimscript)
+  when SupportIoctlInheritCtl:
+    var
+      FIOCLEX {.importc, header: "<sys/ioctl.h>".}: cint
+      FIONCLEX {.importc, header: "<sys/ioctl.h>".}: cint
+
+## Also defined in std/posix and system/io
```

Comment:
sorry I meant regular comment, otherwise it will end in docs (check output of `nim doc`)

Reply:
If we don't want the inheritable flag handling right now (and do it later in another PR), we can remove this part entierely.